ruby/boot.rb

```diff
@@ -0,0 +1,8 @@
+require './config/initializers/database'
+require './app/models/shop'
+require './app/models/product'
+
+shop = Shop.find(1)
+#shop.products.create(title: 'phone', price: 100)
+puts shop.products.to_sql
+
```

Comment:
Please, add a few examples of creating records in the database using all your models